internal/saucecloud/cypress.go

```diff
@@ -36,6 +39,13 @@ func (r *CypressRunner) RunProject() (int, error) {
 		files = append(files, r.Project.RootDir)
 	}
 
+	if r.Project.DryRun {
+		log.Warn().Msg("Running Cypress test in dry run mode.")
```

Comment:
Imho, this is a bit of a misnomer. We aren't running any (cypress) tests. Not even in dry mode. I think we should just print out a generic warning that works for all frameworks that we are running in dry mode.
It's not the _tests_ that are running in a dry mode, but the run command itself. We aren't even remotely interacting with the test framework at this stage. E.g. if cypress itself had a dry mode, this would be 100% confusing.